src/web/tools/TouchEventManager.ts

```diff
@@ -157,7 +156,6 @@ export default class TouchEventManager extends EventManager<HTMLElement> {
       pointerId: event.changedTouches[index].identifier,
       eventType: eventType,
       pointerType: PointerType.TOUCH,
-      buttons: MouseButtons.NONE,
```

Comment:
I'd keep this in ming regarding more than one button. If we were to pass them here, having the type as possibly `undefined` adds one more step when using the API.

If we don't care about multiple buttons at once, then it doesn't matter I guess.

Reply:
I'm not sure what do you mean by that. We are talking about `TouchEventManager` - it won't get any event from mouse. Why would we want to pass them here?

Reply:
I just meant that (assuming we support more than one button at once), passing `0` or `[]` here removes the step of checking if the value is `undefined`.